### DISCUSSION TITLE
I am blocked at step 3 when i want to configurate the homeconnect pluging.

### DISCUSSION THREAD

**Discussion by LeFumiste** [thoukydides/homebridge-homeconnect#302] [Q&A]
Help please !!!

**Comment by thoukydides** [comment]
You need to provide more information than that if you want a useful answer.

For example, which step 3? Do you mean one of these, and if so which?
- Step 3(a) - Homebridge Plugin Installation (recommended approach using Homebridge UI)
- Step 3(b) - Homebridge Plugin Installation (alternative method using command line)

What do you mean by “blocked”? Is there an error message, if so what is it? If not, what is blocking you?

If you are getting a `{ "error": "access_denied", "error_description": "login session expired" }` message when attempting to authorize access via SingleKey ID / Home Connect then change your browser's preferred language to list English first (so that `en` or `en-GB` is the first entry on the `Accept-Language` header). See [#295](https://github.com/thoukydides/homebridge-homeconnect/issues/295#issuecomment-2351038785) for more details.

**Reply by Lamox** [reply]
Sorry for highjacking but I think I have the same problem. The plugin is installed using the UI (3a) on my Synology NAS. Everything looks fine but when I click settings on the plugin I just get a spinning wheel.

This is the log:
Node.js v20.9.0
[10/7/2024, 6:36:04 PM] [Homebridge UI] [homebridge-homeconnect] file:///volume1/homebridge/node_modules/homebridge-homeconnect/dist/settings.js:3
import PACKAGE from '../package.json' with { type: 'json' };
                                      ^^^^

SyntaxError: Unexpected token 'with'
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:118:18)
    at callTranslator (node:internal/modules/esm/loader:273:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:278:30)

**Reply by thoukydides** [reply]
If you are using a Synology NAS then you probably need to remain on v1.0.3 of this plugin until Synology release a sufficiently up-to-date version of Node.js. See #300.

**Reply by Lamox** [reply]
Thanks, I'll install 1.0.3 and take it from there.
